openfisca_france/model/prestations/minima_sociaux/rsa.py

```diff
@@ -648,7 +648,7 @@ class rsa_activite(DatedVariable):
     label = u"Revenu de solidarité active - activité"
     start_date = date(2009, 6, 1)
 
-    @dated_function(start = date(2009, 6, 1), stop = date(2015, 12, 31))
+    @dated_function(start = date(2009, 6, 1))
```

Comment:
@pierrehvz @fpagnoux : we should keep these stp date right ?
ppa started in 2016 and replace RSA activité and we still know nothing of aefa ...